app/src/main/java/org/coepi/android/cen/CENApi.kt

```diff
@@ -14,7 +14,7 @@ interface CENApi {
 
     // get recent keys that have CEN Reports
     @GET("/cenkeys/{timestamp}")
-    fun cenkeysCheck(@Path("timestamp") timestamp : Int): Call<RealmCenKeys>
+    fun cenkeysCheck(@Path("timestamp") timestamp : Int): Call<Array<String>>
```

Comment:
Prefer `List` over `Array`. Unless indexed access/mutation is necessary or performance critical.